ste/xfer-anyToRemote.go

```diff
@@ -335,9 +340,10 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s ISenderBase, si
 		// Final logging
 		if jptm.ShouldLog(pipeline.LogInfo) { // TODO: question: can we remove these ShouldLogs?  Aren't they inside Log?
 			if _, ok := s.(s2sCopier); ok {
-				jptm.Log(pipeline.LogInfo, "COPY SUCCESSFUL")
+				jptm.Log(pipeline.LogInfo, fmt.Sprintf("COPY SUCCESSFUL: %s", strings.Split(info.Destination, "?")[0]))
 			} else if _, ok := s.(uploader); ok {
-				jptm.Log(pipeline.LogInfo, "UPLOAD SUCCESSFUL")
+				// Output relative path of file, includes file name.
+				jptm.Log(pipeline.LogInfo, fmt.Sprintf("UPLOAD SUCCESSFUL: %s", strings.Split(info.Destination, "?")[0]))
```

Comment:
>UPLOAD SUCCESSFUL [](start = 44, length = 17)

consider removing the space in the middle, easier for grepping, since our keywords for failures are: COPYFAILED, DOWNLOADFAILED, UPLOADFAILED

COPYSUCCEEDED, DOWNLOADSUCCEEDED, UPLOADSUCCEEDED

Reply:
@adreed-msft easy grepping is important for our partner teams and support engineers